egs/ami/s5b/local/chain/tuning/run_tdnn_lstm_1l.sh

```diff
@@ -5,11 +5,14 @@
 #System               tdnn_lstm1i_sp_bi_ld5 tdnn_lstm1l_sp_bi_ld5
 #WER on dev        20.6      19.8
 #WER on eval        20.1      19.2
-#Final train prob      -0.045 -0.067
```

Comment:
@danpovey I have emailed the final pdf to you, later will go on updating this PR